pkg/install/install.go

```diff
@@ -140,11 +144,14 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 		api.InstallPhaseDeployStorage: {
 			action(i.createDNS),
 			action(func(ctx context.Context) error {
-				return i.installStorage(ctx, installConfig, platformCreds, image)
+				return i.installStorage(ctx, installConfig)
```

Comment:
I would be for leaving code where it is. Only for a reasons that we should introduce as small change as possible before GA to limit the possibility of breakages. But not my call here :)

Reply:
I'm with @mjudeikis.  The change is absolutely where we want to go but by the time you add the additional db field required, this is going to be a really big PR.  Let's do it piece by piece.  Please revert the pkg/installer restructuring in this PR.